PlayCover/Utils/AlertExtensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  AlertExtensions.swift
+//  PlayCover
+//
+//  Created by Nick on 2022-11-16.
+//
+
+import AppKit
+
+extension NSAlert {
+    static func differentBundleIdKeymap(response: (NSApplication.ModalResponse) -> Void) {
```

Comment:
Personally, I think this function should be in `Keymapping.swift` since it is very specific to keymapping. Also, it might be better to have the function return either `true`/`false` (depending on which button was pressed) instead of relying on it's response function. There can be a variable that then stores the value and handles it appropriately (which will also reduce complexity in the `importKeymap` function)

Reply:
Oh yeah you're right, that would be better than the current implementation.